<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgArrangeMenu</class>
 <widget class="QWidget" name="WdgArrangeMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>174</width>
    <height>275</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Mode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="btnColumnMode">
       <property name="text">
        <string>Column</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="btnRowMode">
       <property name="text">
        <string>Row</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="btnGridMode">
       <property name="text">
        <string>Grid</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>View</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="btnAllView">
       <property name="text">
        <string>All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="btnThumbnailsView">
       <property name="text">
        <string>Thumbnails Only</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="btnCommentsView">
       <property name="text">
        <string>Comments Only</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
